packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -577,7 +577,20 @@ export function signInActor({ services }: SignInMachineOptions) {
           const { user } = context;
           const { confirmation_code } = context.formValues;
 
-          return Auth.verifyTotpToken(user, confirmation_code);
+          const verifyTotp = await Auth.verifyTotpToken(
+            user,
+            confirmation_code
+          );
+          const currentAuth = await Auth.currentAuthenticatedUser();
+          const merged = { ...verifyTotp, ...currentAuth };
```

Comment:
Spreading user object will actually omit all the prototype functions like `user.getSession`: [stackoverflow](https://stackoverflow.com/questions/54164518/does-the-spread-operator-not-copy-over-the-prototype).

Can we just return `Auth.currentAuthenticatedUser()`? Is there any info on `verifyTotp` that's missing on `currentAuth`?

Reply:
Per our meeting this has been reverted back, and now we no longer set user.